engine/runtime/src/main/java/org/enso/interpreter/node/ExpressionNode.java

```diff
@@ -63,7 +79,11 @@ public void setSourceLocation(int sourceStartIndex, int sourceLength) {
    */
   @Override
   public SourceSection getSourceSection() {
-    return EnsoRootNode.findSourceSection(getRootNode(), sourceStartIndex, sourceLength);
+    if (this instanceof ExpressionNodeWrapper wrapper) {
```

Comment:
Probably. But then I don't get this fancy Java pattern matching instanceof. Nevertheless, I would not bother with this code that much, as it will probably be rewritten within a few weeks, once I get to a more proper support for debugging.